### ISSUE TITLE
Flickering LED panel

### ISSUE THREAD

**Issue by tamkar1982** [Howchoo/crypto-ticker#9]
Sometimes the LED's are flickering, do you had that issue too?

**Comment by josephtyler** [comment]
I haven't yet addressed that in this guide, but take a look at @ZLevine's NHL scoreboard guide for an example of how to fix the flicker: https://howchoo.com/pi/diy-raspberry-pi-nhl-scoreboard-led-panel#optional-anti-flicker-mod

**Comment by tamkar1982** [comment]
I have soldered pin 4 and 18 together on the hat and now I have to change the --led-gpio-mapping settings. 
From "adafruit-hat" to "adafruit-hat-**pwm**".
I tried to change it in the "frame.py" file.
Before self.args['led_gpio_mapping'] = kwargs.get('led_gpio_mapping', 'adafruit-hat')
After self.args['led_gpio_mapping'] = kwargs.get('led_gpio_mapping', 'adafruit-hat-pwm')
When I compose and restart the docker than nothing works!
The led are completely off 
Do you have any suggestions how to pass the settings argument otherwise, or is there any other code to check when changing the --led-gpio-mapping settings?

**Comment by josephtyler** [comment]
@ZLevine do you happen to remember when you built the scoreboard if there is anything that looks off here?

**Comment by docjfnoel** [comment]
Did anyone find a solution for the flickering problem?

**Comment by twingall1** [comment]
> Did anyone find a solution for the flickering problem?

Yes, solved mine by following the above article: Soldered wire between pins 4 & 18 on the matrix bonnet. 
Then editing frame.py to make these changes:
led-gpio-mapping=adafruit-hat-pwm
led-slowdown-gpio=2 (not sure if this one helped or not)

Then docker-compose up --build. No more flicker!
